src/main/java/Entities/Property.java

```diff
@@ -82,11 +75,24 @@ public int getPrice(){
     }
 
     public int getRent(){
-        if (this.houses == 0) {
-            return this.rent;
-        } else {
-            return (this.houses * 2) * this.rent;
+        if (this.colour.equals("Railroad")){
+            return 100;
+        } else if (this.colour.equals("Utility")) {
+            return 50;
+        } else if (this.houses==0){
+            return rent;
+        } else if (this.houses==1){
+            return rent1H;
+        } else if (this.houses==2){
+            return rent2H;
+        } else if (this.houses==3){
+            return rent3H;
+        } else if (this.houses==4){
+            return rent4H;
+        } else if (this.houses==5){
+            return rentHotel;
         }
+        return 0; // there was an error if we return 0
```

Comment:
Would it be better if we used a switch block instead?

Reply:
Would probably look cleaner